pkg/operator/ceph/cr_manager.go

```diff
@@ -156,11 +157,19 @@ func (o *Operator) startCRDManager(context context.Context, mgrErrorCh chan erro
 
 	// Set up a manager
 	mgrOpts := manager.Options{
-		LeaderElection:     false,
-		Namespace:          o.config.NamespaceToWatch,
-		MetricsBindAddress: "0",
-		Scheme:             scheme,
-		CertDir:            certDir,
+		LeaderElection: false,
+		// Cache: cache.Options{
+		// 	DefaultNamespaces: map[string]cache.Config{o.config.NamespaceToWatch: {}},
+		// },
+		Metrics: metricsserver.Options{
+			// BindAddress is the bind address for controller runtime metrics server default is 8080. Since we don't use the
+			// controller runtime metrics server, we need to set the bind address 0 so that port 8080 is available.
+			BindAddress: "0",
+		},
+		Scheme: scheme,
+		WebhookServer: webhook.NewServer(webhook.Options{
+			CertDir: certDir,
```

Comment:
If the webhook is not enabled, this won't be an issue, right?

Reply:
no, and I think we are not supporting the webhook currently. So we should be good